ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -6,6 +6,8 @@
 import glob
 import logging
 import shutil
+import sys
+import win32api, win32con
```

Comment:
Can you make this import conditional? Perhaps condition on whether `os.name` is `nt`? I'd prefer avoiding unnecessary imports.

Reply:
Will change.